game/templates/game/landing.html

```
@@ -257,203 +256,434 @@ <h1 class="hero-title">
-                <circle cx="19" cy="19" r="3"/>
-                <circle cx="5" cy="19" r="3"/>
-                <line x1="8.5" y1="5" x2="15.5" y2="5"/>
-                <line x1="8.5" y1="19" x2="15.5" y2="19"/>
-                <line x1="5" y1="8" x2="5" y2="16"/>
-                <line x1="19" y1="8" x2="19" y2="16"/>
-                <line x1="7.1" y1="7.1" x2="16.9" y2="16.9"/>
-                <line x1="7.1" y1="16.9" x2="16.9" y2="7.1"/>
-              </svg>
+            <!-- Dot Indicators -->
+            <div class="about-dots" id="aboutDots">
+                <button class="about-dot active" data-index="0" aria-label="Go to slide 1" aria-pressed="true"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)